storage/backup/backup-service/src/handlers/mod.rs

```diff
@@ -129,3 +130,118 @@ pub(crate) fn get_routes(backup_handler: BackupHandler) -> BoxedFilter<(impl Rep
         }))
         .boxed()
 }
+
+pub(crate) fn get_routes_with_backup_accessor(
```

Comment:
Any context for these changes? They seem slightly orthogonal to me, e.g., different PR? 🤔

Reply:
related. backup service needs to know the sub dbs (event, txn, state store, etc).  When we switch DB based on Fast Sync status, the backup handler service will also need to switch the DB